reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-73.md

```diff
@@ -88,36 +81,36 @@ PowerShell 7.3-preview.8 includes the following features, updates, and breaking
 - Add `find.exe` to legacy argument binding behavior for Windows (#17715)
 - Add `-noprofileloadtime` switch to pwsh (#17535) (Thanks @rkeithhill!)
 
-For a complete list of changes, see the [Change Log][CHANGELOG] in the GitHub repository.
+For a complete list of changes, see the [Change Log][11] in the GitHub repository.
 
 ## Experimental Features
 
 PowerShell 7.3 introduces the following experimental features:
 
-- [PSExec][exp-psexec] - Adds the new `Switch-Process` cmdlet (alias `exec`) to provide `exec`
-  compatibility for non-Windows systems. In PowerShell 7.3-preview.8, this feature became
+- [PSExec][05] - Adds the new `Switch-Process` cmdlet (alias `exec`) to provide `exec`
+  compatibility for non-Windows systems. In PowerShell 7.3-rc.1, this feature became
   mainstream.
-- [PSCleanBlock][exp-clean] - Adds `clean` block to script block as a peer to `begin`, `process`,
-  and `end` to allow easy resource cleanup. In PowerShell 7.3-preview.8, this feature became
+- [PSCleanBlock][04] - Adds `clean` block to script block as a peer to `begin`, `process`,
+  and `end` to allow easy resource cleanup. In PowerShell 7.3-rc.1, this feature became
   mainstream.
-- [PSStrictModeAssignment][exp-strict] - Adds the **StrictMode** parameter to `Invoke-Command` to
-  allow specifying strict mode when invoking command locally. In PowerShell 7.3-preview.8, this
+- [PSStrictModeAssignment][07] - Adds the **StrictMode** parameter to `Invoke-Command` to
+  allow specifying strict mode when invoking command locally. In PowerShell 7.3-rc.1, this
   feature was removed.
-- [PSNativeCommandErrorActionPreference][exp-error] - Adds the
+- [PSNativeCommandErrorActionPreference][06] - Adds the
   `$PSNativeCommandUseErrorActionPreference` variable to enable errors produced by native commands
   to be PowerShell errors.
-- [PSAMSIMethodInvocationLogging][exp-amsi] - Extends the data sent to AMSI for inspection to
-  include all invocations of .NET method members. In PowerShell 7.3-preview.8, this feature became
+- [PSAMSIMethodInvocationLogging][02] - Extends the data sent to AMSI for inspection to
+  include all invocations of .NET method members. In PowerShell 7.3-rc.1, this feature became
   mainstream.
-- Remove [PSNativePSPathResolution][exp-path] experimental feature.
+- Remove [PSNativePSPathResolution][03] experimental feature.
 
-For more information about the Experimental Features, see [Using Experimental Features][exp].
+For more information about the Experimental Features, see [Using Experimental Features][01].
 
 ## Breaking Changes and Improvements
 
 - `Test-Connection` is broken due to an intentional
-  [breaking change](https://github.com/dotnet/runtime/issues/66746) in .NET 7. It's tracked by
-  [#17018](https://github.com/PowerShell/PowerShell/issues/17018)
+  [breaking change][09] in .NET 7. It's tracked by
+  [#17018][10]
```

Comment:
I can't leave a suggestion that crosses edited and non-edited lines, but:

```markdown
- `Test-Connection` is broken due to an intentional [breaking change][09] in .NET 7. It's tracked
  by [#17018][10]
```

Reply:
fixed